src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url("/fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.search-form {
+  position: relative;
+}
+
+.search-input {
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  outline: none;
+  color: #3d4e61;
+  font-family: Avenir-Heavy, sans-serif;
+  font-size: 16px;
+}
+
+.search-input::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  color: #3d4e61;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-input:focus {
+  background: linear-gradient(180deg, #fff 0, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.big-search-input {
+  padding: 23px 0 23px 62px;
+}
+
+.big-search-input::placeholder {
+  font-size: 16px;
+}
```

Comment:
put it in the .search-input, delete styles for .big-search-input, leave styles for .small-search-input
(checklist point 2)